src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final int DATE = 0;
+    static final int NAME = 1;
+    static final int WORKING_HOUR = 2;
+    static final int INCOME_PER_HOUR = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int[] perHour = new int[names.length];
+        String[] splitedString = new String[4];
+        StringBuilder stringBuilder = new StringBuilder(
+                "Report for period " + dateFrom + " - " + dateTo);
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
Formatter should be a constant as it is pointed out in the common mistakes, string of that format can also be another constant BTW

Reply:
i got, thx. fixed